table/policy.go

```diff
@@ -3619,8 +3627,14 @@ func (r *RoutingPolicy) GetAllPolicy() []*config.PolicyDefinition {
 	r.mu.RLock()
 	defer r.mu.RUnlock()
```

Comment:
Don't defer - place the unlock immediately after reading the map. Line 3634 in this case.

Reply:
Thanks, I'll unlock it after reading it.